vms/evm/contract_message_test.go

```diff
@@ -7,44 +7,12 @@ import (
 	"encoding/hex"
 	"testing"
 
-	"github.com/ava-labs/avalanchego/ids"
 	"github.com/ava-labs/avalanchego/utils/logging"
-	"github.com/ava-labs/avalanchego/vms/platformvm/warp"
 	"github.com/ava-labs/awm-relayer/config"
-	warpPayload "github.com/ava-labs/subnet-evm/warp/payload"
-	"github.com/ethereum/go-ethereum/common"
 	"github.com/stretchr/testify/assert"
 	"go.uber.org/mock/gomock"
 )
 
-// Used to create a valid unsigned message for testing. Should not be used directly in tests.
-func createUnsignedMessage() *warp.UnsignedMessage {
```

Comment:
Let's keep function. For context, it's used to generate a valid unsigned warp message, which can be hardcoded as the expected value in `TestUnpack`. If the warp message format changes, then `TestUnpack` should fail, signalling that we should make corresponding changes in `awm-relayer`. We can remove this function once the warp message format is stable.

Reply:
sure. since it causes the unused function linter error, I will try to update the config so that it will not error on this.

Reply:
I think you can use `//nolint:unused` above the function signature and it should let the linter pass

Reply:
fixed.